junit5-jupiter-starter-maven/pom.xml

```diff
@@ -35,10 +35,6 @@
 
 	<build>
 		<plugins>
-			<plugin>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.8.1</version>
-			</plugin>
```

Comment:
❌ This causes Maven to use a more recent version of the maven-compiler-plugin than declared in its super POM. AFAIK that's a "best practice" and should be kept.

Reply:
@marcphilipp I agree in general, that explicitly overriding plugin versions from the super POM can be a good practice.

But I think the sample projects should be kept small and only use configuration actually needed for Junit 5. Currently, the Maven starter declares two plugins:
- The maven-surefire-plugin which is essential for Junit 5. Without this declaration, the tests won't run
- The maven-compiler-plugin which is not needed for Junit 5

I think a casual reader will not understand the difference. That's why I propose to remove the declaration of the maven-compiler-plugin.

Reply:
@sormuras WDYT?

Reply:
As long as the sample projects to compile and test successfully, I'm fine with pruning not (longer) required configuration details. IIRC, some of the samples failed to build when using the default m-c-p ... but that might be outdated (and depend on the version of Maven being used).